### PR TITLE
Add Mock API

### LINKED ISSUE #19: Create a mock API to start integrating our components into an app
As we already have a nice set of components, we can start integrating them into an app. But, as we don't have a backend yet, we're going to create a mock API. You can search what's a mock API is. I've just briefly read [this post](https://www.soapui.org/learn/mocking/what-is-api-mocking/) and it looks fine.

As this will be the first time for you doing this kind of task, I'll create a couple of tickets related to the mock API, in order to implement something small first, and to improve it ticket by ticket.

So, our first approach will be a simple "instantaneous" API to request all the items "from the server". You have to create the file `src/api/index.js` and in it, you have to implement the function `getFiles()` which will do the magic (ie, it will return all the items in the "from the server").

Regarding the items, you simply have to create a static list of items in the same file, and that's the list that will be returned. And how should and item look like? Well, it should be an object containing three properties for now, an `image` (the URL of the image), a `title` (a string) and a `price` (a number representing the price in USD of the item). Yes, basically the same props that our [`ItemList`](https://github.com/franc5/store/blob/main/src/components/ItemList/index.js#L19-L23) component accepts.

Feel free to ask any question, I may have omitted some important details.

### PR DESCRIPTION
**Pull request by magarciaprieto**
Closed #19 .

In an aside meeting it was determined that the function's name will be changed to getItems() (previously: getFiles() ). The number of items in the array of objects was set to 5. 

Cheers boss!

### CODE REVIEW

**Comment by magarciaprieto** [comment]
Sorry about the milestones in this PR hehe. I forgot to save some changes.

Tschüss!

File: src/index.js [hunk 1]
```diff
@@ -8,6 +9,7 @@ ReactDOM.render(
     <div>
       Placeholder App
     </div>
+  < MockApi />
```
Comment (franc5) [inline src/index.js:12]: `MockApi` isn't a React component, it's just VanillaJS. Adding this here breaks the app. Let's have a conversation on a side to properly discuss this.

**Review by franc5 — CHANGES_REQUESTED** [review]
Please, address [this comment](https://github.com/franc5/store/pull/23/files#r583277559).

File: src/index.js (hunk 1 above)
Comment (magarciaprieto) [reply src/index.js:12]: This was written before our last meeting. I'll delete it right now, sorry.

**Comment by magarciaprieto** [comment]
All changes requested were addressed. Closed #19 via #23

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import MockApi from './api'
```
Comment (franc5) [inline src/index.js:3]: We can avoid importing `MockApi` here. Let's delete this line in a separated PR.

**Review by franc5 — APPROVED** [review]

**Pull request merged**